src/NumericalAlgorithms/LinearOperators/PowerMonitors.hpp

```diff
@@ -45,4 +51,80 @@ std::array<DataVector, Dim> power_monitors(const DataVector& input_data_vector,
                                            const Mesh<Dim>& mesh);
 /// @}
 
+/*!
+ * \ingroup SpectralGroup
+ * \brief Returns the maximum value of a variable in the element.
+ *
+ * Compute the maximum of variable \f$ u \f$ in the element as
+ *
+ * \f{align*}{
+ *  u_\mathrm{max} = \max_{ijk} \left\lvert u_{ijk} \right\rvert ,
+ * \f}
+ *
+ * where \f$ u_{ijk}\f$ are the nodal coefficients
+ * of variable \f$ u \f$.
+ *
+ */
+double maximum_of_variable(const DataVector& input_data_vector);
+
+/// @{
+/*!
+ * \ingroup SpectralGroup
+ * \brief Relative truncation error.
+ *
+ * Truncation error according to Eqs. (57) and (58) of Ref.
+ * \cite Szilagyi2014fna, i.e.,
+ *
+ * \f{align*}{
+ *  \mathcal{T}\left[P_k\right] = \log_{10} \max \left(P_0, P_1\right)
```

Comment:
I have yet to remove the log10's from the power monitors function. I'll leave them here for now since I'm not sure about changing the formulas form the Szilagyi paper/dg-charm doc

Reply:
Corrected the description